lib/validators.ts

```diff
@@ -27,6 +27,8 @@ SOFTWARE.
 ************************************************************************************ */
 "use strict";
 
+const toString = Object.prototype.toString
```

Comment:
The `toString` used in this file (L54) is this method, accessed via the global object. That's weird, and TS complains.